src/buyingcatalogue/Chart.yaml

```diff
@@ -29,11 +29,11 @@ dependencies:
   version: ~0.1.0
 - name: bapi-db-deploy
   condition: bapi-db-deploy.enabled
-  version: 1.24.9
+  version: 1.24.12
```

Comment:
We wouldn't usually update the core component versions outside of a release

Reply:
I'll revert this change